src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -25,9 +29,22 @@ public FindCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
-        String[] nameKeywords = trimmedArgs.split("\\s+");
+        String[] allKeywords = trimmedArgs.split("\\s+");
 
-        return new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+        List<String> nameKeywords = Arrays.asList(allKeywords);
+
+        List<String> tagKeywords = new ArrayList<>();
+        for (String keyword : allKeywords) {
```

Comment:
This way of parsing keywords and tags is a little inconsistent with other parsers. But it's ok to leave it as it is now I think.

Reply:
Yep, I agree with you. In my initial implementation I faced problems using the `ArgumentMultimap` which other parsers use as some keywords do not have a prefix (those that we want to search the task names instead of tags), hence decided to use this implementation for now. But I think I can try to improve on this later on! Thanks!!